Kubernetes-Practical/Service/Service.yaml

```
@@ -0,0 +1,85 @@
+  name: nginx-service-loadbalancer
+spec:
+  selector:
+    app: nginx
+ports:
+- protocol: TCP
+  port: 80
+  targetPort: 80
+type: LoadBalancer  
+
+
+### Headless Service 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)